Client/lib/gMap.js

```diff
@@ -0,0 +1,231 @@
+
+var gMap = function(){
+
+  getGeolocation(startGMap);
+  
+  var lastLocationClicked;
+  var markerCords = [];
+  var travelButtonList = [];
+  var travelMode = 'WALKING';
+  var map;
+
+  function startGMap(pos){
+    //can be -> BICYCLING, DRIVING, TRANSIT, WALKING
+
+    //----------------------------------
+    //creatig the Map
+    //----------------------------------
+    var mapOptions = {
+      center: pos,
+      zoom: 12,
+      backgroundColor: '#568',
+      disableDoubleClickZoom : true,
+      disableDefaultUI: true,
+      keyboardShortcuts: false
+    };
+  	map = new google.maps.Map(document.getElementById('gMap'),mapOptions);
```

Comment:
map is declared on line further up for wider scope of the other functions but I will defiantly correct the indentation issue.